apps/content/services/font.py

```
@@ -0,0 +1,234 @@
+
+    def create_font(
+        self,
+        *,
+        publisher_id: int,
+        name_ar: str | None,
+        name_en: str | None,
+        description_ar: str | None,
+        description_en: str | None,
+        long_description_ar: str | None,
+        long_description_en: str | None,
+        license: LicenseChoice,
```

Ruff (0.15.20)

[error] Function argument `license` is shadowing a Python builtin

(A002)